src/site/content/en/blog/compat2021-holiday-update/index.md

```diff
@@ -0,0 +1,150 @@
+([permalink](https://wpt.fyi/results/?q=label%3Ainterop-2021&run_id=5629744973348864&run_id=5117500464300032&run_id=5734588178497536)) showing all of the tests included in Compat 2021, 
+and also views for [Chrome](https://wpt.fyi/results/?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5738932147847168&run_id=5629744973348864), 
+[Firefox](https://wpt.fyi/results/?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5644039006191616&run_id=5117500464300032), 
+and [Safari](https://wpt.fyi/results/?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5739124314079232&run_id=5734588178497536) comparing the results to our last update in July.
+
+Let’s take a look at the improvements in each area!
+
+
+## CSS flexbox
+
+`flex-basis: content` is now on its way to all browsers, with implementations landing in 
+[Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=470421) and 
+[WebKit](https://trac.webkit.org/changeset/284440/webkit). (The `content` value was already 
+supported by Firefox.)
+
+In Chromium, an [issue with flexbox sizing](https://bugs.chromium.org/p/chromium/issues/detail?id=961902) 
+is fixed, matching the spec and Firefox’s behavior. And in Gecko, several 
+[issues affecting Compat 2021](https://bugzilla.mozilla.org/show_bug.cgi?id=1700745) are fixed, 
+including an [issue with percentage height on flex items](https://bugzilla.mozilla.org/show_bug.cgi?id=1611303). 
+Finally, in WebKit, support for more alignment property values ([left, right](https://trac.webkit.org/changeset/282078/webkit),
+[self-start, self-end](https://trac.webkit.org/changeset/282267/webkit), [start, end](https://trac.webkit.org/changeset/281840/webkit)) 
+is now added, and a lot of improvements were made for [absolute positioning](https://trac.webkit.org/changeset/281995/webkit), 
+also improving the flexbox test results in Compat 2021.
+
+
+## CSS Grid
+
+The use of CSS Grid on the web continues to grow, as can be seen in both the 
+[2021 Web Almanac](https://almanac.httparchive.org/en/2021/css#flexbox-and-grid-adoption) and 
+Chrome’s [usage metrics](https://www.chromestatus.com/metrics/feature/timeline/popularity/1693).
+
+The [launch of GridNG](https://blogs.windows.com/msedgedev/2021/08/10/compat2021-css-grid-gridng/) 
+in Chrome and Edge 93 resolved many long standing issues with Grid, closing an impressive 38 issues 
+in Chromium’s bug tracker. Together with many smaller improvements that followed, the Compat 2021 
+score for Grid in Chromium improved by 3% to 97%. This work was led by the Edge team at Microsoft.
+
+An [absolute positioning bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1707643) affecting Grid 
+was fixed in Gecko, and [many fixes](https://bugs.webkit.org/buglist.cgi?bug_status=RESOLVED&email1=zsun%40igalia.com&emailassigned_to1=1&emailtype1=equals&list_id=7730965&query_format=advanced&resolution=FIXED&short_desc=grid&short_desc_type=allwordssubstr)
```

Comment:
Maybe instead https://bugs.webkit.org/buglist.cgi?bug_status=RESOLVED&chfield=resolution&chfieldfrom=2021-01-01&chfieldto=2021-12-31&component=Accessibility&component=CSS&component=Layout%20and%20Rendering&component=New%20Bugs&f1=short_desc&f2=short_desc&f3=short_desc&list_id=7736912&o1=notsubstring&o2=notsubstring&o3=substring&query_format=advanced&resolution=FIXED&v1=Web%20Inspector&v2=%5BLFC%5D&v3=grid (which has quite a lot more fixes!)

Reply:
Thanks, let's go with that!